source/material_model/interface.cc

```diff
@@ -440,221 +440,220 @@ namespace aspect
       }
 
 
-      namespace
```

Comment:
It gave me some errors with the explicit instaniation (it didn't like to instantiate something in an anonymous namespace. I thought about giving it a name, but I didn't really see the use of this namespace so I removed it.